docker/tests/binaries.json

```diff
@@ -0,0 +1,40 @@
+{
+    "released": [
+        "tezos-smart-rollup-client-Proxford",
+        "tezos-smart-rollup-client-PtNairob",
+        "tezos-smart-rollup-node-Proxford",
+        "tezos-smart-rollup-node-PtNairob",
+        "tezos-baking",
+        "tezos-sapling-params",
+        "tezos-accuser-Proxford",
```

Comment:
Add `tezos-smart-rollup-wasm-debugger` here and rebase please, after that I'll approve.